### ISSUE TITLE
Problem with importing i18next

### ISSUE THREAD

**Issue by Vrajmas** [i18next/react-i18next#571]
Hi, i came across a problem
i'm using 
TypeScript 3.1.3 on react 
"i18next": "^11.9.1"
"react-i18next": "^8.1.0"
 "@types/react-i18next": "^7.8.2"

Problem is " node_modules/@types/i18next/index has no default export. " on import i18next in  node_modules/react-i18next/index.d.ts file 

Current code 
```
import i18next, {
  TranslationFunction,
  ReactOptions as I18nOptions,
} from 'i18next';
```

i made some changes 
```
import  {
  TranslationFunction,
  ReactOptions as I18nOptions,
} from 'i18next';
import * as i18next from 'i18next';
```
 And it worked for me. Can somebody help me with this problem

**Comment by beheh** [comment]
Should be fixed #567.

**Comment by Vrajmas** [comment]
The pull PR will be today ?
